app/packs/controllers/autosubmit_controller.js

```diff
@@ -22,6 +22,31 @@ export default class extends Controller {
     }, 0)
   }
 
+  capacitySubmit(event) {
+    clearTimeout(this.timeout)
+
+    this.timeout = setTimeout(() => {
+      var min_capacity = parseInt(this.min_capacityTarget.value)
+      var max_capacity = parseInt(this.max_capacityTarget.value)
+
+      if (min_capacity > max_capacity) {
+        this.capacity_errorTarget.classList.add("capacity-error--display")
```

Comment:
It is possible that this class is already present in the ClassList. Do we need some conditional check or ruby takes care of it ?

Reply:
The span for the error is on the page, and it's hidden by default:
`<span class="capacity-error--hide text-red-700 text-sm mt-2 -mb-2" data-autosubmit-target="capacity_error">`

capacity-error--hide class in application.sass:
`.capacity-error--hide
  @apply hidden`

The autosubmit_controller.js is changing this class to [capacity-error--display] to display the error if there is an error, or to [capacity-error--hide] if there is no error. 

Please let me know if my explanations are not clear and if you have more questions.

Reply:
As per follow up discussion with Rita, the class gets added only once even though it is already present inn the list.